src/Tasks/Microsoft.NET.Build.Tasks/DependencyContextBuilder.cs

```diff
@@ -1,759 +0,0 @@
-// Copyright (c) .NET Foundation and contributors. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Security.Cryptography;
-using System.Text;
-using Microsoft.Extensions.DependencyModel;
-using NuGet.Packaging;
-using NuGet.Packaging.Core;
-using NuGet.ProjectModel;
-using NuGet.Versioning;
-
-namespace Microsoft.NET.Build.Tasks
-{
-    internal class DependencyContextBuilder
```

Comment:
Now that this is gone, can we rename `DependencyContextBuilder2` to `DependencyContextBuilder`?

Reply:
I figured it would be easier to review without the rename.  But now I've pushed a commit that does rename it.